public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -217,6 +221,7 @@ export const RulesGroup = React.memo(({ group, namespace, expandAll, viewMode }:
             </div>
           </>
         )}
+        {isFolderBulkActionsEnabled && folderUID && isListView && <FolderBulkActionsButton folderUID={folderUID} />}
```

Comment:
It looks like the current solution is mainly designed for removing imported rules. I'm wondering if we've considered adding this action to the Alert rules tab within the folders instead? This might not only limit the scope of changes required but also prevent the need for bulk actions on the alert list page, which could simplify future maintenance

Reply:
But thinking about it, I believe it makes more sense to place the bulk action in the alerting list view, as that’s where users naturally expect to manage alerting entities. Also, I don’t really see how this other approach improves maintainability.